plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/engine/openai/OpenAIModels.java

```diff
@@ -45,6 +40,9 @@ private OpenAIModels() {
         new AIModel("o1-pro", 200_000, Set.of(AIModelFeature.CHAT)),
         new AIModel("o1", 200_000, Set.of(AIModelFeature.CHAT)),
         new AIModel("o1-mini", 128_000, Set.of(AIModelFeature.CHAT)),
+        new AIModel("gpt-5", 400_000, Set.of(AIModelFeature.CHAT)),
```

Comment:
Do we really need to hardcode these models?
Chat feature should be automatically discoverable from model ID.